code/game/machinery/suit_storage_unit.dm

```diff
@@ -195,6 +194,16 @@
 	storage = null
 	occupant = null
 
+/obj/machinery/suit_storage_unit/emp_act()
+	. = ..()
+	uv_super ^= TRUE
```

Comment:
```suggestion
	uv_super = TRUE
```

Reply:
I made it toggle it instead of setting it to true like you suggested, what's wrong with this?

Reply:
in that case, this is how it's usually done
```suggestion
	uv_super = !uv_super
```